packages/web-components/src/components/content-block-cards/__stories__/content-block-cards.stories.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
+import ArrowRight20 from 'carbon-web-components/es/icons/arrow--right/20';
+import ifNonNull from 'carbon-web-components/es/globals/directives/if-non-null.js';
+import { select } from '@storybook/addon-knobs';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+import readme from './README.stories.mdx';
+import '../content-block-cards';
+import '../../content-block/content-block-heading';
+import '../../card-group/card-group';
+import '../../card-group/card-group-item';
+import '../../cta/card-cta-footer';
+import '../../cta/video-cta-container';
+
+import { CTA_TYPE } from '../../cta/shared-enums';
+
+const ctaTypes = {
+  [`Local (${CTA_TYPE.LOCAL})`]: CTA_TYPE.LOCAL,
+  [`Jump (${CTA_TYPE.JUMP})`]: CTA_TYPE.JUMP,
+  [`External (${CTA_TYPE.EXTERNAL})`]: CTA_TYPE.EXTERNAL,
+};
+
+const cardGroupItem = html`
+  <dds-card-group-item href="https://example.com">
+    <div slot="heading">Nunc convallis lobortis</div>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit.
+      Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
+    </p>
+    <dds-card-cta-footer slot="footer">
+      ${ArrowRight20({ slot: 'icon' })}
+    </dds-card-cta-footer>
+  </dds-card-group-item>
+`;
+
+const cardGroupItemWithImages = html`
+  <dds-card-group-item href="https://example.com">
+    <dds-image
+      slot="image"
+      alt="Image alt text"
+      default-src="https://fpoimg.com/1056x792?text=4:3&amp;bg_color=ee5396&amp;text_color=161616"
+    >
+    </dds-image>
+    <div slot="eyebrow">Topic</div>
+    <div slot="heading">Natural Language Processing.</div>
+    <dds-card-cta-footer slot="footer">
+      ${ArrowRight20({ slot: 'icon' })}
+    </dds-card-cta-footer>
+  </dds-card-group-item>
+`;
+
+const cardGroupItemWithVideos = html`
+  <dds-card-group-item cta-type="video" href="1_9h94wo6b">
+    Test Video - 1:1
```

Comment:
Does something like https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/13b0181/packages/web-components/src/components/content-group-simple/__stories__/content-group-simple.stories.ts#L192 help?